tests/unittests/CMakeLists.txt

```diff
@@ -1,3 +1,11 @@
+if (MSVC)
```

Comment:
This seems like the result of not building googletest

Reply:
llvm also have googletest and test target